ui/src/components/Publication/Creation/LinkedItems/index.tsx

```diff
@@ -0,0 +1,169 @@
+                    },
+                    user.token
+                );
+            } catch (err) {
+                setError('There was a problem removing the topic.');
+            }
+            fetchAndSetLinks(user.token, 'TOPIC');
+        }
+    };
+
+    // When making a research problem, we refer to "items" to link to (because it could be a topic), and not just "publications"
+    const isProblem = type === 'PROBLEM';
+    const linkableEntityLabel = isProblem ? 'item' : 'publication';
+
+    return (
+        <div className="space-y-6 lg:space-y-10 2xl:w-10/12">
+            <div>
+                <Components.PublicationCreationStepTitle
+                    text={`What ${linkableEntityLabel}s should this publication be linked from?`}
+                    required
+                />
+                <p className="mb-6 block text-sm text-grey-800 transition-colors duration-500 dark:text-white-50">
+                    All publications in Octopus are linked to each other to form research chains, branching down from
+                    research problems to real world implementations.{' '}
+                    {isProblem &&
+                        ' For research problems, if there is no existing publication on the system that yours relates to, you can link to a research topic instead.'}
+                </p>
+
+                <p className="mb-6 block text-sm text-grey-800 transition-colors duration-500 dark:text-white-50">
+                    Your {Helpers.formatPublicationType(type)} must be linked from at least one other{' '}
+                    {linkableEntityLabel} on Octopus. {Helpers.formatPublicationType(type)} can be linked from{' '}
+                    {availableLinkTypes.map((type, index) => {
+                        return `${Helpers.formatPublicationType(type)}${
+                            index !== availableLinkTypes.length - 1 ? ', ' : isProblem ? ' and Research Topic. ' : '. '
+                        }`;
+                    })}
+                    For more information on publication types, please see the{' '}
+                    <Components.Link href="/faq#pub_type_octopus">FAQ</Components.Link>. The search box below has been
+                    filtered to only show {linkableEntityLabel}s that can be linked to your publication type.{' '}
+                    {isProblem && ' Use the dropdown to switch between searching for publications or research topics.'}
+                </p>
+
+                <p className="mb-6 block text-sm text-grey-800 transition-colors duration-500 dark:text-white-50">
+                    This approach structures the content of the platform and makes your work more discoverable. It also
+                    helps you avoid repeating work, as there is no need to keep re-writing descriptions of well-known
+                    Research Problems, or Methods etc.
+                </p>
+            </div>
+
+            <div className="relative">
```

Comment:
This layout needs some additional css based on ticket description no. 4: _"To the right of the dropdown, a search is present. The selected item type in the dropdown determines what item type will display in the search box’s results preview"_

The UI should look something like this from what I understand:
![image](https://github.com/JiscSD/octopus/assets/48569671/a5c4f328-a3a3-4514-ab13-a90661cbc211)

But it looks like this:
![image](https://github.com/JiscSD/octopus/assets/48569671/5aeeca90-15b7-4efc-b107-c83d0f479bdd)

Reply:
Fixed the position now with your help!